tests/test_wellstate.cpp

```diff
@@ -440,6 +441,9 @@ BOOST_AUTO_TEST_CASE(TESTWellContainer) {
     BOOST_CHECK_EQUAL(wc.well_name(1), "W2");
     BOOST_CHECK_THROW(wc.well_name(10), std::exception);
 
+    const auto& wells = wc.wells();
+    std::vector<std::string> expected = {"W1", "W2"};
+    BOOST_CHECK( std::is_permutation( wc.begin(), wc.end(), expected.begin(), expected.end());
```

Comment:
Looks like there's a right parenthesis missing in the call to `BOOST_CHECK` here.